template.php

```diff
@@ -478,13 +478,21 @@ function campaignion_foundation_payment_forms_payment_form_alter(&$element, \Pay
  * Implements hook_webform_component_render_alter().
  */
 function campaignion_foundation_webform_component_render_alter(&$element, $component) {
-  // Add extra classes for donation amount radios.
-  if ($element['#webform_component']['form_key'] == 'donation_amount') {
+  // Add donation button classes for various form keys.
+  $form_key = $element['#webform_component']['form_key'];
+  if (substr($form_key, 0, strlen('donation_amount')) === 'donation_amount') {
     if (in_array($element['#type'], ['radios', 'select_or_other'])) {
       $element['#wrapper_attributes']['class'][] = 'donation-amount';
       $element['#attributes']['class'][] = 'donation-amount-buttons';
     }
   }
+  $form_key = $element['#webform_component']['form_key'];
+  if (substr($form_key, 0, strlen('donation_interval')) === 'donation_interval') {
+    if (in_array($element['#type'], ['radios'])) {
+      $element['#wrapper_attributes']['class'][] = 'donation-interval';
+      $element['#attributes']['class'][] = 'donation-interval-buttons';
+    }
+  }
```

Comment:
I'm not sure how useful it is to set a button class on the donation interval field per default? It shouldn’t break anything when there are no styles for the class, but do you really think we need it?